Project.toml

```diff
@@ -3,5 +3,9 @@ uuid = "88a69b33-da0f-4502-8c8c-d680cf4d883b"
 authors = ["Luiz M. Faria"]
 version = "0.1.0"
 
+[deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
```

Comment:
I am `OK` with depending on `LinearAlgebra`, but I doubt we really need `StaticArrays` as a dependency.